samples/simulation/ising/trotter-evolution/Program.qs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.Ising {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation RunTrotter() : Unit {
```

Comment:
The Trotter decomposition is the special case of the lowest-order Trotter–Suzuki decomposition; we've generally been far too lax about that distinction, but it may be good to reflect the correct terminology in the entry point at least?

```suggestion
    operation RunTrotterSuzuki() : Unit {
```